### PR TITLE
Add deprecation notice

### PR DESCRIPTION
**Pull request by jimthematrix**
Signed-off-by: Jim Zhang <jim.zhang@kaleido.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,14 @@ Licensed under Creative Commons Attribution 4.0 International License
 https://creativecommons.org/licenses/by/4.0/
 -->
 
+# DEPRECATION NOTICE
+
+**NOTE**: This project is currently in `deprecated` state. Depending on your needs, please refer to one of the following projects instead:
+
```
Comment (arsulegai) [inline README.md:9]: The closest project matching privacy-preserving off-chain smart contract execution following similar principles is in labs https://github.com/hyperledger-labs/private-data-objects

File: README.md [hunk 2]
```diff
@@ -32,9 +40,10 @@ confidentiality for code and data, our initial implementation uses
 Intel® Software Guard Extensions (SGX).
 
 Hyperledger Avalon leverages the existence of a distributed ledger to
- * Maintain a registry of the trusted workers (including their attestation info)
- * Provide a mechanism for submitting work orders from a client(s) to a worker
- * Preserve a log of work order receipts and acknowledgments
+
+- Maintain a registry of the trusted workers (including their attestation info)
+- Provide a mechanism for submitting work orders from a client(s) to a worker
+- Preserve a log of work order receipts and acknowledgments
```
Comment (shemnon) [inline README.md:47]: Are these markdown style changes really needed?  It makes the "blame" searching harder to follow.

File: README.md [hunk 3]
```diff
@@ -59,19 +68,22 @@ See the [contributing document](CONTRIBUTING.md)
...
-* Yevgeniy Y. Yarmosh (EugeneYYY)
+
+- Manjunath A C (manju956)
+- Daniel Anderson (danintel)
+- Thomas J Barnes (TomBarnes)
+- Srinath Duraisamy (srinathduraisamy)
+- Manoj Gopalakrishnan (manojgop)
+- Karthika Murthy (Karthika)
+- Ramakrishna Srinivasamurthy (ram-srini)
+- Yevgeniy Y. Yarmosh (EugeneYYY)
```
Comment (shemnon) [inline README.md:79]: And here, do we really need to do `*` -> `-`?

File: README.md [hunk 4]
```diff
@@ -59,19 +68,22 @@ See the [contributing document](CONTRIBUTING.md)
...
 
 ## Sponsor
-* Mic Bowman (cmickeyb) - TSC member
+
+- Mic Bowman (cmickeyb) - TSC member
```
Comment (shemnon) [inline README.md:84]: Finally here as well, do we really need to do `*` -> `-`?

File: README.md [hunk 5]
```diff
@@ -59,19 +68,22 @@ See the [contributing document](CONTRIBUTING.md)
...
 
 ## License
+
```
Comment (shemnon) [inline README.md:86]: Is this a style issue or a markdown validity issue?  If the former is it needed?

File: README.md (hunk 2 above)
Comment (petermetz) [reply README.md:47]: ^^ +1

File: README.md [hunk 6]
```diff
@@ -3,6 +3,15 @@ Licensed under Creative Commons Attribution 4.0 International License
...
 -->
 
+# DEPRECATION NOTICE
+
+**NOTE**: This project is currently in `deprecated` state. Depending on your needs, please refer to one of the following projects instead:
+
+- for core programming in enclave based confidential computation, [Confidential Computing Consortium](https://confidentialcomputing.io/)
+- for using Zero Knowledge Proofs in privacy preserving blockchain transactions, [Hyperledger Ursa](https://github.com/hyperledger/ursa)
+- for achieving privacy using offchain data channels with blockchain coordination, [Hyperledger FireFly](https://github.com/hyperledger/firefly)
+- for similar funtionality, see Hyperledger [lab Private Data Objects](https://github.com/hyperledger-labs/private-data-objects)
```
Comment (tkuhrt) [inline README.md:13]:
```suggestion
- for similar funtionality, see [Private Data Objects](https://github.com/hyperledger-labs/private-data-objects), a Hyperledger Lab
```

**Comment by tkuhrt** [comment]
LGTM

**Pull request merged**